### ISSUE TITLE
Text inputs not optimized in dark mode

### ISSUE THREAD

**Issue by markkarrica** [LunarMarathon/graphToTable#47]
**Describe the bug**
When switching to dark mode, text inputs are not readable (because of the lack of color contrast).

**To Reproduce**
1. Go to '[Graph to table](https://graphtotable.netlify.app/)'
2. Write any text inside any text input
3. Switch to dark mode (if you're not already)
4. The text inside the inputs isn't readable

**Expected behavior**
I expect to read clearly the text inside the text inputs even in dark mode.

**Screenshots**
![image](https://user-images.githubusercontent.com/60916242/195948820-7fe022cc-1981-4c56-9f0e-1e440dace115.png)
![image](https://user-images.githubusercontent.com/60916242/195948841-3c7ffd47-c9f4-4ad8-bbbd-264be70ac470.png)

**Desktop (please complete the following information):**
 - Browser: Google Chrome (106.0.5249.119)

**Comment by markkarrica** [comment]
Also,
I would like to actively contribute to this issue.
Please, could you assign this issue to me?

**Comment by LunarMarathon** [comment]
I'm sorry, I had already noticed it yesterday while merging dark mode Pr and UI rework one. There were similar issues with the autocomplete dropdown and checked state bg as well. All of these along with few more changes have been committed. 

Please check other open issues if you'd like to contribute, thank you
